platform/app/public/config/google.js

```diff
@@ -55,6 +55,9 @@ window.config = {
         supportsWildcard: false,
         dicomUploadEnabled: true,
         omitQuotationForMultipartRequest: true,
+        cloudServerAPI: {
+          typeId: 'ohif.cloudServerAPI.google',
```

Comment:
lets call it id

Reply:
probably makese sense to rename it to configurationAPI or similar too

Reply:
Agreed and as a result I have renamed ALL related classes and functions.